### DISCUSSION TITLE
Outdated and Poorly Translated Mirror Site Found on Bing

### DISCUSSION THREAD

**Discussion by Ovler-Young** [nuejs/nue#244] [General]
While my friend is searching for "nue" on Bing with Chinese as the preferred language, she stumbled upon a site called "[nue.dev](https://nue.dev)". This site appears to be an outdated and poorly machine-translated version of the official Nue.js website.

The translation quality is extremely poor, and it could easily mislead and confuse people who come across it, especially non-English speakers who rely on translated content to learn and use Nue.js.

I think it would be beneficial to consider adding a warning on the official Nue.js website and documentation about this mirror site. This way, users can be cautious and avoid any confusion or misinformation. Alternatively, we could explore the possibility of reporting the site to Bing to prevent it from appearing in search results.

**Comment by tipiirai** [comment]
WOW. Someone really made a huge effort to that! I'd like to contact the author to do the official translation once the content settles. Any way to find the contact information?

**Reply by Ovler-Young** [reply]
hmmmm. Seems that site is down, the domain owner didn't renew the domain……
